app/src/main/res/drawable/ic_add_text_button.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
maybe a better name? This is the image of the floating action button right? maybe it is better to say in the name that is a button not to add just text, but instead a tweet right?¿